cabal.project

```diff
@@ -0,0 +1,13 @@
+packages: .
+
+-- why 0.3 is not in mokus0 tree?
```

Comment:
https://github.com/mokus0/dependent-map/pull/26 - should be merged and released shortly. I'll remove this override as soon as that's done.

Reply:
[It's up](https://hackage.haskell.org/package/dependent-map-0.3)